src/Framework/Models/Factories/ModelFactory.php

```diff
@@ -85,16 +98,31 @@ public function create(array $attributes = [])
         return $this->count === 1 ? $instances[0] : $instances;
     }
 
+    /**
+     * @unreleased
+     */
+    public function createAndResolveTo($property)
```

Comment:
Looks like we can add the `Closure` return type here, too!

Reply:
Good catch. I added the `Closure` to this method as well, see [2aa34ce](https://github.com/impress-org/givewp/pull/7626/commits/2aa34cedeecb683ad5d3cb936b38c05ed8268280).